code/handlers/trackservers.q

```diff
@@ -118,12 +118,13 @@ cleanup:{if[count w0:exec w from`.servers.SERVERS where not .dotz.livehn w;
 
 / add a new server for current session 
 addnthawc:{[name;proctype;hpup;attributes;W;checkhandle]
+    if[ not 0="J"$last ":"vs string hpup ;
```

Comment:
Could you invert the condition here & just return early?
```suggestion
    if[0="J"$last ":"vs string hpup ;:()];
```

Reply:
Yep, can do. Is there a benefit of doing this instead?

Reply:
I think it's easier to follow - as a general rule, I prefer shorter conditionals. With a long multi-line conditional it's easier to miss where the conditional ends & get confused about what code is executed conditionally.